std_msgs/Header header
float64 cam_x_dist_reader         # cam_x_dist_reader reading
float64 cam_x_dist_reader_var     # cam_x_dist_reader_var reading

